invenio/modules/knowledge/testsuite/test_knowledge_restful.py

```diff
@@ -172,13 +172,7 @@ def test_get_knwkb_not_exist(self):
             user_id=1,
         )
 
-        answer = get_answer.json
-
-        expected_result = dict(
-            status=404,
-        )
-
-        assert answer['status'] == expected_result['status']
+        assert get_answer.status_code == 404
```

Comment:
because, for some reasons, devel and release requirements suffers also about of another problem: https://s3.amazonaws.com/archive.travis-ci.org/jobs/71936871/log.txt
```python
ERROR: test_get_knwkb_not_exist (invenio.modules.knowledge.testsuite.test_knowledge_restful.TestKnowledgeRestfulAPI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/inveniosoftware/invenio/invenio/modules/knowledge/testsuite/test_knowledge_restful.py", line 181, in test_get_knwkb_not_exist
    assert answer['status'] == expected_result['status']
KeyError: 'status'
```